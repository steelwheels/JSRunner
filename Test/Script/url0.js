
const url0 = URL("tmp_dir/a") ;
if(isNull(url0)){
	console.print("url0 = <null>\n") ;
} else if(isUndefined(url0)){
	console.print("url0 = <undefined>\n") ;
} else {
	console.print("url0 = " + url0.absoluteString + "; \n") ;
}

