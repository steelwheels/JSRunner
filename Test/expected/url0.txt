url0 = file:///Users/tomoo/Project/JSTools/OSX/tmp_dir/a; 
